notebooks/MAST/exploring_jwst_transmission/comparing_spectral_types_jwst.ipynb

```diff
@@ -0,0 +1,7074 @@
+{
```

Comment:
<p>You should have some sort of introduction, even if it's similar to the other notebook. Each notebook should be "standalone", so don't worry about being having similar content in two notebooks. Consider: what astronomy background should people be familiar with to do this? Summarize and link to sources. What is your goal in this notebook? To compare features of an F-type star to G-type star, so let's state that here.</p><p> </p><p>I agree with [link previously written SpectralDB notebook] :)</p><p>Once you work out the rest of introduction, you might want to add something like "For more information on using Specviz and SpectralDB", see the notebook [link]. I'll make sure it gets a link when we publish them all</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/ScienceBetter/notebooks-stsci/pull/5/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='4'/>